store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1684,6 +1698,54 @@ private void doUnlockMailbox(String[] args) throws ServiceException {
             }
         }
     }
+    
+    private void doCreateHabOrgUnit(String[] args)   {
+        try {
+        if(args.length != 3) { 
+            usage();
+            return;
+        }
+        Domain domain = lookupDomain(args[1], prov, Boolean.FALSE);
+        
+        if (!(prov instanceof SoapProvisioning)) {
```

Comment:
since, required methods are already added in LdapProvisioning. we can support -l option too.

Reply:
Fixed